src/renderer/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -55,13 +60,23 @@ const Dashboard: React.FunctionComponent<IOwnProps &  Pick<IRootState, "auth">>
     };
 
     const onRemoveValidator = (index: number): void => {
-        // delete locally from array
-        const v = [...validators];
-        v.splice(index, 1);
+        setSelectedValidatorIndex(index);
+        setConfirmModal(true);
+    };
+
+    const onConfirmDelete = async (): Promise<void> => {
+        const validatorsData = props.auth.auth;
+        if(validatorsData){
+            const validators =validatorsData.getValidators();
+            const selectedValidatorPublicKey = validators[selectedValidatorIndex].publicKey.toHexString();
+            const selectedV4Keystore = new V4Keystore(
+                path.join(validatorsData.directory,selectedValidatorPublicKey + ".json"));
```

Comment:
this does not work, it leaves "/" at the end of "selectedValidatorPublicKey" string
"Uncaught (in promise) Error: Cannot find file " ...
file ends with 
"...0aa22ec11cc/.json" 
instead of "0aa22ec11cc.json"

Reply:
oh yeah,  sorry, you are right